webapp-django/crashstats/crashstats/models.py

```diff
@@ -1239,11 +1256,17 @@ class RawCrash(SocorroMiddleware):
     )
 
     def get(self, **kwargs):
-        format = kwargs.get('format', 'meta')
-        if format == 'raw_crash':
-            format = kwargs['format'] = 'raw'
-        kwargs['expect_json'] = format != 'raw'
-        return super(RawCrash, self).get(**kwargs)
+        fmt = kwargs.get('format', 'meta')
```

Comment:
``fmt`` confused me for a bit. How about ``format_`` or ``data_type`` or ``crash_format``?